File: app/src/main/java/com/owncloud/android/ui/fragment/GalleryFragmentBottomSheetActions.java
package com.owncloud.android.ui.fragment;

public interface GalleryFragmentBottomSheetActions {

    /**
     * hide all the images in particular Folder.
     */
    void hideImages(boolean isHideImagesClicked);

    /**
     * hide all the videos in particular folder.
     */
    void hideVideos(boolean isHideVideosClicked);

    /**
     * load all media of a particular folder.
     */
    void selectMediaFolder();

}

File: app/src/main/java/com/owncloud/android/ui/fragment/GalleryFragmentBottomSheetActions.kt
/*
 * Nextcloud Android client application
 *
 * @author TSI-mc
 * Copyright (C) 2022 TSI-mc
 * Copyright (C) 2022 Nextcloud GmbH
 *
 * SPDX-License-Identifier: AGPL-3.0-or-later OR GPL-2.0-only
 */

package com.owncloud.android.ui.fragment

interface GalleryFragmentBottomSheetActions {
    /**
     * show/hide all the images & videos in particular Folder.
     */
    fun updateMediaContent(mediaState: GalleryFragmentBottomSheetDialog.MediaState)

    /**
     * load all media of a particular folder.
     */
    fun selectMediaFolder()
}
